client/settings/fraud-protection/advanced-settings/cards/cvc-verification.js

```diff
@@ -13,37 +13,47 @@ import FraudProtectionRuleCard from '../rule-card';
 import FraudProtectionRuleDescription from '../rule-description';
 import FraudProtectionRuleCardNotice from '../rule-card-notice';
 
-const CVCVerificationRuleCard = () => (
-	<FraudProtectionRuleCard
-		title={ __( 'CVC Verification', 'woocommerce-payments' ) }
-		description={ __(
-			'This filter checks the security code submitted by the customer against the data on file with the card issuer.',
-			'woocommerce-payments'
-		) }
-	>
-		<FraudProtectionRuleDescription>
-			{ __(
-				'Because the card security code appears only on the card and not on receipts or statements, the card security code ' +
-					'provides some assurance that the physical card is in the possession of the buyer.',
+const CVCVerificationRuleCard = () => {
+	const declineOnCVCFailure =
+		window.wcpaySettings?.accountStatus?.fraudProtection
```

Comment:
Showing different messages according to the CVC check settings on the platform, its design is [here](https://www.figma.com/file/WIM5w2Y10SCNBFdMLSVftj/%5BWCPay%5D-Fraud-%26-Risk-Tools?node-id=2930-263372&t=V91lulvTyXqJtCTK-0).